tests/Common/BranchComponentTest.php

```diff
@@ -1463,21 +1463,4 @@ public function testConfigurationRollback()
             'currentVersion',
         ]);
     }
-
-    /**
-     * @param string $branchPrefix
-     */
-    protected function deleteBranchesByPrefix(DevBranches $devBranches, $branchPrefix)
```

Comment:
Neni potreba, vyresi BranchWorkspacesTest

Reply:
Klidně to mohl být trait ale whatever.